modules/nodes-cluster-enabling-features-install.adoc

```diff
@@ -0,0 +1,89 @@
+// Module included in the following assemblies:
+//
+// * nodes/cluster/nodes-cluster-enabling-features.adoc
+
+:_content-type: PROCEDURE
+[id="nodes-cluster-enabling-features-install_{context}"]
+= Enabling feature sets at installation
+
+You can enable feature sets for all nodes in the cluster by editing the `install-config.yaml` file before you deploy the cluster.
+
+.Prerequisites
+
+* You have an `install-config.yaml` file.
+
+.Procedure
+
+. Use the `featureSet` parameter to specify the the name of the feature you want to enable. For example, `TechPreviewNoUpgrade`.
++
+.Sample `install-config.yaml` file with an enabled feature set
+
+[source,yaml]
+----
+compute:
+- hyperthreading: Enabled
+  name: worker
+  platform:
+    aws:
+      rootVolume:
+        iops: 2000
+        size: 500
+        type: io1
+      metadataService:
+        authentication: Optional
+      type: c5.4xlarge
+      zones:
+      - us-west-2c
+  replicas: 3
+featureSet: TechPreviewNoUpgrade
+----
+
+. Save the file and reference it when using the installation program to deploy the cluster.
+
+[NOTE]
+====
+Enabling the `TechPreviewNoUpgrade` feature set cannot be undone and prevents minor version updates. These feature sets are not recommended on production clusters.
+====
+
+.Verification
+
+You can verify that the feature gates are enabled by looking at the `kubelet.conf` file on a node after the cluster is installed.
+
+. Start a debug session for a node:
++
+[source,terminal]
+----
+$ oc debug node/<node_name>
+----
+
+. Change your root directory to the host:
++
+[source,terminal]
+----
+sh-4.2# chroot /host
+----
+
+. View the `kubelet.conf` file:
++
+[source,terminal]
+----
+sh-4.2# cat /etc/kubernetes/kubelet.conf
+----
++
+.Sample output
++
+[source,terminal]
+----
+ ...
+featureGates:
+  TechPreviewNoUpgrade: true,
```

Comment:
Just checking whether this is supposed to have a comma on the end or not.

Reply:
@adellape Excellent catch.
@patrickdillon Just want to double check but I think this got by us. It should be `true` correct?

Reply:
@mburke5678 I picked up this verification steps from the existing content. Do you know if the sample output should be `true,` or `true`?

Reply:
@adellape Just to tie this off. Mike is going to test this on his side. If an update is required, he plans on being in this file for something else and can update the sample output then.